cfy_manager/components/prometheus/prometheus.py

```diff
@@ -133,8 +134,34 @@ def replace_certificates(self):
             _handle_certs()
             service.reload(PROMETHEUS, append_prefix=False,
                            ignore_failure=True)
-            service.restart(BLACKBOX_EXPORTER, append_prefix=True,
+        if exists(constants.NEW_INTERNAL_CA_CERT_FILE_PATH):
+            self._handle_blackbox_exporter_ca()
```

Comment:
Will this code be executed on Manager nodes only?  The `blackbox_exporter` is installed solely there and does not exist on db/broker nodes https://github.com/cloudify-cosmo/cloudify-manager-install/blob/56ba5c73108ccdb69034a72e974facb9da122ebd/cfy_manager/components/prometheus/prometheus.py#L70-L76

Reply:
Yep, NEW_INTERNAL_CA_CERT is only copied to manager instances